lib/Doctrine/ORM/Exception/NotSupported.php

```diff
@@ -10,4 +10,9 @@ public static function create(): self
     {
         return new self('This behaviour is (currently) not supported by Doctrine 2');
     }
+
+    public static function createForDbal3(): self
+    {
+        return new self('This behaviour is not supported by doctrine/dbal:3.x, please use doctrine/dbal:2.x instead.');
```

Comment:
I think we should improve this exception message, because it gives no indiication on how to get to DBAL 3 making it sound DBAL 2 is better. Maybe:

```suggestion
        return new self('Feature was deprecated in doctrine/dbal 2.x and is not supported by installed doctrine/dbal:3.x, please see the doctrine/deprecations logs for new alternative approaches.');
```

Reply:
Thanks @beberlei! Sounds much better :)